pynest/nest/tests/test_tsodyks2_connection.py

```diff
@@ -26,72 +26,71 @@
 
 @nest.ll_api.check_stack
 class Tsodyks2ConnectionTest(unittest.TestCase):
-    """
-    Functional test for the "tsodyks2" synapse: compare NEST implementation to
+    '''
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#string-quotes) triple-quoted docstrings should always use double quotes.

```suggestion
    """
```